pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -115,6 +116,15 @@ in stdenv.mkDerivation rec {
     if doInstallCheck then "${arrow-testing}/data" else null;
   PARQUET_TEST_DATA =
     if doInstallCheck then "${parquet-testing}/data" else null;
+  GTEST_FILTER =
+    if doInstallCheck then let
+      # Upstream Issue: https://issues.apache.org/jira/browse/ARROW-11398
+      filteredTests = lib.optionals stdenv.hostPlatform.isAarch64 [
+        "TestFilterKernelWithNumeric/3.CompareArrayAndFilterRandomNumeric"
+        "TestFilterKernelWithNumeric/7.CompareArrayAndFilterRandomNumeric"
+        "TestCompareKernel.PrimitiveRandomTests"
```

Comment:
Tests still fail. Also, I suppose that "3." and "7." may be RNG-dependent. I would just mark aarch64 as broken for now.

Reply:
There was an extra `'` in the constructed filter variable. I corrected that.
The numbers should be fixed, please run it again.

I don't think we should block arrow for the entire arch because a test for a pretty advanced feature fails.